lib/import-export-visitor.js

```diff
@@ -186,6 +210,13 @@ module.exports = class ImportExportVisitor extends Visitor {
       const bodyInfo = this.bodyInfos[i];
       const parts = [];
 
+      // We don't need to add a "use strict" directive unless the compiler
+      // made any changes.
+      if (this.madeChanges &&
+          bodyInfo.needToAddUseStrictDirective) {
+        parts.push('"use strict";');
```

Comment:
the `.bodyInfos` is reify specific metadata right? Is there a better way to differentiate what is legit AST from what is custom metadata. I noticed while refactoring sometimes I had to take an educated guess/leap.